protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
```

Comment:
At some point we discussed all this (to be able to store STRs in an array and ease the lookup).  I can't find a pointer to it though.  Maybe it was only on hangout.  We should get better at documenting these things that diverge from the paper.